substrate/frame/examples/split/Cargo.toml

```
@@ -20,32 +20,18 @@ targets = ["x86_64-unknown-linux-gnu"]
 codec = { workspace = true }
 log = { workspace = true }
 scale-info = { features = ["derive"], workspace = true }
+frame = { workspace = true, features = ["experimental", "runtime"]}
```

```suggestion
frame = { workspace = true, features = ["runtime"]}
```
Remove `experimental` feature flag